### DISCUSSION TITLE
Is there any place to get BrowserWindow instance?

### DISCUSSION THREAD

**Discussion by christlee1989** [eclipse-theia/theia#8495] [General]
In Electron Main, Is there any place to get BrowserWindow instance? ElectronMainApplication don’t seem to reveal BrowserWindow instance after createWindow

**Comment by paul-marechal** [comment]
`ElectronMainApplication.createWindow` returns a `Promise<BrowserWindow>`. It should reveal, not sure what is wrong with your setup.
